### PR TITLE
Ignored Spark TMP Client & Custom Discs Data, add Copper to Anti X-Ray and update Paper.

### PR DESCRIPTION
**Pull request by 105hua**
Title says it all.

### CODE REVIEW

**Comment by 105hua** [comment]
Another commit was added to resolve conflicts.

**Review by Kiddooo — APPROVED** [review]
LGTM 👍

File: .gitignore [hunk 1]
```diff
@@ -74,3 +74,8 @@ paper.yml-README.txt
 .HARD_FAILED
 .NO_UNPATCH
 .NO_PULL
+
+plugins/spark/tmp-client/
+plugins/CustomDiscs/musicdata
+plugins/CustomDiscs/data.yml
+world/datapacks/copper-door-cutting/LICENSE
```
Comment (Prof-Bloodstone) [inline .gitignore:81]: NIT: This file won't be created automatically, so you could just delete it. But regardless, we should keep the license.

File: plugins/ProtocolLib/lastupdate
Comment (Prof-Bloodstone) [inline plugins/ProtocolLib/lastupdate:1]: Do we need this file? I see that on prod it has different value - so it'll cause conflict. IMO - should be gitignored and removed.

**Review by Prof-Bloodstone — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (105hua) [reply .gitignore]: Retracked custom discs data and copper door cutting license, which should solve this.

File: plugins/ProtocolLib/lastupdate
Comment (105hua) [reply plugins/ProtocolLib/lastupdate:1]: Yeah, it shouldn't be required on the repo itself, I ignored and untracked it.

**Comment by 105hua** [comment]
Appropriate changes have been made.

**Review by Prof-Bloodstone — APPROVED** [review]
:shipit: LGTM

**Pull request merged**